.github/workflows/test.yaml

```diff
@@ -6,7 +6,6 @@ on:
       - "**.go"
     branches:
       - main
-
```

Comment:
intentional diff?

Reply:
lint formatter. Should I revert it?

Reply:
i.e. it was made automatically? odd... IMO breaking up the major sections with a newline helps readability, but this is a small file -- I think it's ok to keep.

For the future I'd recommend either (a) disabling it, or (b) opening a PR to apply its recommendations (so we don't end up with more stray changes like this :smile:)